sgf-viewer/src/content.ts

```diff
@@ -114,7 +133,7 @@ export const pages: object =
                     "/nas/ucb/norabelrose/kellin-analysis-game-fixed.sgf"
                 ],
                 "max_games": 10,
-                "adversary": "498 million training steps, 600 visits",
+                "adversary": "545 million training steps, 600 visits",
```

Comment:
Did we change this game (doesn't look like it, and we'd probably have to redo the analysis if we did...)? Probably shouldn't change the # of training steps then.